guide/resources/wallet-interoperability.md

```diff
@@ -0,0 +1,196 @@
+
+Here are some common scenarios:
+- Using different wallets for desktop and mobile
+- Paying someone who uses different software
+- Migrating to a different device, operating system or application
+- Multi-signature wallets that require multiple devices and applications to exchange data
+- Using your own node
+
+Each of these requires that applications are designed and developed to be open, ideally relying on standardized formats.
+
+So here are some ways to allow users to more seamlessly navigate different software configurations.
+
+## Wallet import and export
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/key-import-export.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/key-import-export@2x.jpg"
+   alt-text = "Transfer of wallet data between applications"
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Allow for wallets generated in one application to be easily restored in another application. Over the years, Bitcoin applications have implemented various technical details in different ways, partly because standards take time to evolve. See [Wallets Recovery](https://walletsrecovery.org) as an illustration of the problem.
+
+</div>
+
+## Payment links
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/payment-links.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/payment-links@2x.jpg"
+   alt-text = "Click a link to launch Bitcoin software"
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Avoid copying & pasting payment data by providing clickable links with the right information embedded. Bitcoin applications can respond to such link clicks by automatically navigating to the payment screen and prefilling the provided form fields. Payment links as defined in [BIP 21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) can contain contain address, amount, label and other details.
+
+Example link:
+```
+bitcoin:175tWpb8K1S7NmH4Zx6rewF9WQrcZv245W?amount=50&label=Luke-Jr&message=Donation%20for%20project%20xyz
+```
+
+</div>
+
+## QR codes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/qr-code.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/qr-code@2x.jpg"
+   alt-text = "A smartphone camera scanning a QR code"
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+QR codes (Quick Response codes) are visual representations of data. Since most devices today feature cameras with built-in support for reading QR codes, this technique has become a convenient method to transfer data from one device to another, even if those devices are offline. Common use cases include reading payment invoices (such as the payment links described above), or importing wallet keys from a backup.
```

Comment:
Personal opinion. But I think more people know what  a QR code is than Quick Response Code, no need to have this, maybe it's just me :smile: 

```suggestion
QR codes are visual representations of data. Since most devices today feature cameras with built-in support for reading QR codes, this technique has become a convenient method to transfer data from one device to another, even if those devices are offline. Common use cases include reading payment invoices (such as the payment links described above), or importing wallet keys from a backup.
```